clustered/client/src/test/java/org/ehcache/clustered/client/CacheManagerDestroyTest.java

```diff
@@ -124,14 +125,15 @@ public void testDestroyCacheManagerDoesNotAffectsExistingCacheWithExistingClient
 
     assertThat(cache.get(1L), is("One"));
 
+    persistentCacheManager2.close();
```

Comment:
I tried it and I find it makes the code more complex to read for little added value. I want the `close()` to happen when all goes well and pretty much do not care when stuff goes wrong ... because that should only happen locally for a dev. Makes sense?